serde/src/java/org/apache/hadoop/hive/serde2/AbstractSerDe.java

```diff
@@ -90,7 +90,7 @@ public void initialize(Configuration configuration, Properties tableProperties,
     Preconditions.checkArgument(this.columnNames.size() == this.columnTypes.size(),
         "Column names must match count of column types");
 
-    log.debug("SerDe initialized: [{}][{}]", this.configuration, this.properties);
+    log.debug("{} initialized: [{}][{}]", getClass().getName(), this.configuration, this.properties);
```

Comment:
For the log message here, I think it makes sense to add `getClass().getName()`.

In the sub-classes though it may be cleaner if we just add a `private static final Logger` field and get done with it. This may also be insignificantly more efficient saving the call to `getClass().getName()`.

Reply:
Also I left the getClass().getName() in AbstractEncodingAwareSerDe.java as it is again an abstract class.